mqtt/mqtt-broker/src/broker.rs

```diff
@@ -302,7 +302,11 @@ where
                     session.send(event)?;
                 }
 
+                let subscriptions =
```

Comment:
nit: move this line with variable initialization closer to usage, please. Now it is 1 line farther than it should be

Reply:
borrow checker rules prevent to move this line closer to publish method